src/daemon/__DAEMON_PACKAGES__

```diff
@@ -1,5 +1,6 @@
 \
           sharutils \
+          lsof \
```

Comment:
I believe `fuser` is available by default so can't we use it instead?

Reply:
I found more interesting information in lsof